requirements.txt

```diff
@@ -9,6 +9,7 @@ werkzeug
 # Optional
 psycopg2
 sqlalchemy
+twine
```

Comment:
That's a good point! 👍 I just pushed a command to check for the ``twine`` cmd and error out if it cannot be found.

I wouldn't like scripts to install dependencies automatically. The risk of not having a virtualenv active is maybe a bit too high. 🙊